src/firefly/js/metaConvert/vo/ObsCoreConverter.js

```diff
@@ -211,6 +211,7 @@ function getObsCoreRowMetaInfo(table,row) {
 function createObsCoreTitle(table,row) {
  // 1. try a template
     const template= getAppOptions().tapObsCore?.productTitleTemplate;
+    if (typeof template !== 'undefined' && template.trim() === '') return ''; //if template is an empty string, assume that's what user wants and return empty string as is
```

Comment:
template can be undefined and you to use it.

use: WebUtil - isDefined
```js
isDefined(getAppOptions().tapObsCore?.productTitleTemplate) 
```

Reply:
ok, after you slack message, I responded to just use lodash trim(), but after even more thought...

```js
if (!(template??'').trim()) return '';
```
should be fine.